x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferenceFeatures.java

```diff
@@ -25,7 +26,8 @@ public Set<NodeFeature> getFeatures() {
         return Set.of(
             TextSimilarityRankRetrieverBuilder.TEXT_SIMILARITY_RERANKER_RETRIEVER_SUPPORTED,
             RandomRankRetrieverBuilder.RANDOM_RERANKER_RETRIEVER_SUPPORTED,
-            SemanticTextFieldMapper.SEMANTIC_TEXT_SEARCH_INFERENCE_ID
+            SemanticTextFieldMapper.SEMANTIC_TEXT_SEARCH_INFERENCE_ID,
+            SemanticQueryBuilder.SEMANTIC_TEXT_INNER_HITS
```

Comment:
Question - should we leverage [getHistoricalFeatures()](https://github.com/elastic/elasticsearch/blob/c65fc630bdbe70b1387e18ab136a5d9091614d58/server/src/main/java/org/elasticsearch/features/FeatureSpecification.java#L57) here instead of `getFeatures()` for this particular case?